LICENSE

```diff
@@ -645,14 +645,14 @@ the "copyright" line and a pointer to where the full notice is found.
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Update all copyright notices with HTTPS links, as mentioned in the license here (with this change, and according to the newest version on gnu.org), the copyright notices in files should have HTTPS links.

Reply:
I think the copyright notice should be 

```
/*
 * Copyright (C) <year-file-created> - <year, up to date> Team NewPipe <tnp@newpipe.schabi.org>
 * <file>.<extension> is part of NewPipe.
 *
 * NewPipe is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * NewPipe is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with NewPipe.  If not, see <https://www.gnu.org/licenses/>.
 */
```
change NewPipe by NewPipe Extractor for NewPipe Extractor.

And move this right after package declaration, before imports